src/features/cell-renderers/loadingRowRenderer.js

```diff
@@ -0,0 +1,19 @@
+// @flow
```

Comment:
We should have a different folder for this. This is a rowRenderer and not a cellRenderer